l10n_hr_account_base/models/account_move.py

```diff
@@ -90,6 +90,13 @@ def _auto_init(self):
                 WHERE (state = 'posted' AND name != '/' AND move_type IN ('out_invoice', 'out_refund'));
             """)
 
+    @api.depends('move_type', 'company_id')
+    def _compute_is_ref_required(self):
+        for rec in self:
```

Comment:
@brunoFilipGithub 
zbog citljivosti uvijek napisi u for-u kroz sto tocno iterrias
```suggestion
        for move in self:
```